pages/providers.vue

```diff
@@ -66,8 +66,15 @@ definePageMeta({
   routeName: "providers",
 });
 
-useHead({
+useSeoMeta({
   title: "Qiskit Providers",
+  ogTitle: "Qiskit Providers",
+  description:
+    "All the ways you can run Qiskit with! from Local Simulators to real Quantum Hardware",
```

Comment:
this sentence need to be updated. Maybe something like:

> _All the ways you can run Qiskit! From Local Simulators to real Quantum Hardware_

Reply:
thanks!